asset-bundle-converter/Assets/AssetBundleConverter/AssetBundleConverter.cs

```diff
@@ -84,7 +84,7 @@ public AssetBundleConverter(Environment env, ClientSettings settings)
 
             log.verboseEnabled = true;
 
-            ShaderGraphMaterialGenerator.k_ShaderPathPrefix = "Assets/git-submodules/glTFast/Runtime/Shader/";
+            // ShaderGraphMaterialGenerator.k_ShaderPathPrefix = "Assets/git-submodules/glTFast/Runtime/Shader/";
```

Comment:
If you pull you should be able to revert this change

Reply:
ah yeah, you're right, I forgot